File: firstcode/firstassignemt/myjavaassignment/LargestTwoNumbers.java
import java.util.Scanner;
public class LargestTwoNumbers {
	public static void main(String[] args) {
		System.out.println("Welcome friend");
	
	Scanner input =  new Scanner(System.in);
	int firstLargest = 0;
	int secondLargest = 0;
	int number = 0;
	int counter = 1;
	
	while(counter <= 10) {
		System.out.print("enter a number");
			number = input.nextInt();
		if(firstLargest < number) {
			firstLargest = number;
		}		
		else if(firstLargest > secondNumber) {
			secondLargest = number;
		}
	counter = counter + 1;
}
	System.out.println(firstLargest);
	System.out.println(secondLargest);
}
}

File: firstcode/firstassignemt/newassignement/largesttwonumbers.py
first_largest = 0
second_largest = 0
for number in range (1, 11):
	number = int(input("Enter a number: "))
	current_number = number
	
	if first_largest < current_number:
		first_largest = current_number
	elif second_largest < number or current_number < number:
		second_largest = number	

print(first_largest) 
print(second_largest)
